code/modules/projectiles/updated_projectiles/guns/pistols.dm

```diff
@@ -112,8 +112,8 @@
 	recoil_unwielded = CONFIG_GET(number/combat_define/min_recoil_value)
 
 /obj/item/weapon/gun/pistol/m1911/custom
-	name = "\improper M1911A1 Custom pistol"
-	desc = "A 20th century military firearm that received several modifications. It seems to have been lovingly taken care of and passed down the family. Lacks an auto magazine eject feature"
+	name = "\improper Colt Government Mk. IV Series 100"
+	desc = "(M1911A1) A CGMkIV100 Colt custom pistol. It lacks an auto magazine eject feature, so it's not recommended for beginners. Crafted by an Armsdealer's Concord manufacturer that was almost all but forgotten."
```

Comment:
CGMkIV100 would be a good one for spelling bee, but horrible for this game. Also the Concord armsdealer addendum makes descriptions simply more pedantic, consider moving the unnecessary fluff to the equivalent codex files if possible (and make sure there is no incongruence between the two).